kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/InstancesController.java

```diff
@@ -0,0 +1,15 @@
+package dev.ikm.komet.kview.mvvm.view.pattern;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ListView;
+import org.carlfx.cognitive.loader.InjectViewModel;
+import org.carlfx.cognitive.viewmodel.SimpleViewModel;
+
+public class InstancesController {
+    @InjectViewModel
+    private SimpleViewModel simpleViewModel;
```

Comment:
you could change this to be the patternViewModel 

`@InjectViewModel
private PatternViewModel patternViewModel`

and match it with what is in the config down below

Reply:
Yes we could do this in the other ticket. Nice suggestion. We could share the parent's view model. My only issue, is if the observable list of instances is very large (1M+) it might not be a good idea to store it. (this is basically read-only rows of data) i think :-).